types/chart.js/index.d.ts

```diff
@@ -26,14 +26,18 @@
 declare class Chart {
     static readonly Chart: typeof Chart;
     constructor(
-        context: string | CanvasRenderingContext2D | HTMLCanvasElement | ArrayLike<CanvasRenderingContext2D | HTMLCanvasElement>,
-        options: Chart.ChartConfiguration
+        context:
```

Comment:
@ElianCordoba should these formatting be reverted?

Reply:
Yes please, all of them. Beside that the change looks good

Reply:
I've reverted all of those prettier changes on this PR so that only the weight property is on this PR.

Would you like me to add those prettier formatting changes on a separate PR or just ignore them altogether?

Reply:
As DT maintainer I'd like to see the prettier changes eventually, but I'll defer to @ElianCordoba and @ricardo-mello.